5.5/root/usr/share/container-scripts/mysql/lib.sh

```diff
@@ -0,0 +1,3 @@
+function info-log {
```

Comment:
`common_functions.sh` or `morecommon.sh` are just weird. I like `lib.sh` best, because it kind of is a "library".